src/ledger/parse/ledger.js

```diff
@@ -8,6 +8,11 @@ import type {GetLedger} from '../types.js';
 function parseTransactionWrapper(ledgerVersion, tx) {
   const transaction = _.assign({}, _.omit(tx, 'metaData'),
     {meta: tx.metaData});
+
+  // normalize format from rippled
+  transaction.meta.delivered_amount = transaction.meta.DeliveredAmount;
```

Comment:
Maybe we can add a comment indicating this is a bug in rippled. The rippled docs say this is deprecated.

> DeliveredAmount	String or Object	DEPRECATED. Replaced by delivered_amount. Omitted if not a partial payment.

Otherwise, LGTM